app/graphql/mutations/users/update_user_details_mutation.rb

```diff
@@ -0,0 +1,20 @@
+module Mutations
+  module Users
+    class UpdateUserDetailsMutation < Mutations::BaseMutation
```

Comment:
Can we add a spec for this, covering the three use cases in the associated form, including an error case.

Here's a sample, should you need it:

https://github.com/oddballio/magnifier/blob/master/spec/graphql/mutations/statistics/update_statistic_mutation_spec.rb